composer.json

```diff
@@ -32,6 +33,6 @@
     },
     "require-dev": {
         "phpunit/phpunit": "9.5.9",
-        "google/gax": "1.9.0"
+        "google/gax": "dev-compute-lro as 1.10"
```

Comment:
This will be removed when the changes in https://github.com/googleapis/gax-php/pull/337 are released.